state/memcached/memcached.go

```diff
@@ -102,10 +103,32 @@ func getMemcachedMetadata(metadata state.Metadata) (*memcachedMetadata, error) {
 	return &meta, nil
 }
 
+func (r *Memcached) parseTTL(req *state.SetRequest) (int32, error) {
```

Comment:
Return *int32 and make it a difference if TTL is set vs value is 0.

Reply:
Requested changes completed with #1011